core/domain/exp_domain.py

```diff
@@ -3177,6 +3177,65 @@ def _convert_states_v18_dict_to_v19_dict(cls, states_dict):
 
         return states_dict
 
+
+    @classmethod
+    def _convert_states_v20_dict_to_v21_dict(cls, states_dict):
+        """Converts from version 20 to 21. Version 21 converts non textangular
+        html content to the textangular html content.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+        for state_dict in states_dict.values():
+
```

Comment:
Remove this line.

A lot of the comments made for 4985 transfer over to here, and should be addressed similarly.